app/src/main/java/com/findmyrecycling/MainActivity.kt

```diff
@@ -35,19 +40,29 @@ fun RecycleSearch(name: String) {
     var location by remember{ mutableStateOf("")}
 
     Column {
-        OutlinedTextField(
+        TextField(
```

Comment:
I found that using the TextField gave more flexibility with the desired modifications like background color

Reply:
Ahhh, are you sure?  It looks like the same function list to me.

https://developer.android.com/reference/kotlin/androidx/compose/material/package-summary#TextField(androidx.compose.ui.text.input.TextFieldValue,kotlin.Function1,androidx.compose.ui.Modifier,kotlin.Boolean,kotlin.Boolean,androidx.compose.ui.text.TextStyle,kotlin.Function0,kotlin.Function0,kotlin.Function0,kotlin.Function0,kotlin.Boolean,androidx.compose.ui.text.input.VisualTransformation,androidx.compose.foundation.text.KeyboardOptions,androidx.compose.foundation.text.KeyboardActions,kotlin.Boolean,kotlin.Int,androidx.compose.foundation.interaction.MutableInteractionSource,androidx.compose.ui.graphics.Shape,androidx.compose.material.TextFieldColors)